.github/actions/display-jest-test-summary/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Display Jest Test Summary
+
+inputs:
+  test_report_path:
+    type: string
+    description: Path of test report JSON file
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Transform Input
+      id: transform_input
+      env:
+        INPUT_PATH: ${{ inputs.test_report_path }}
+      run: |
+        echo JSON_OUTPUT=$(  jq '{ numTotalTests, numPassedTests, numFailedTests}' $INPUT_PATH ) >> $GITHUB_ENV
+      shell: bash
+    - name: Display Jest Test Summary
+      uses: actions/github-script@v6
+      env:
+        JSON: ${{ steps.transform_input.outputs.JSON_OUTPUT }}
+      with:
+        script: |
+          const input = JSON.parse(process.env.JSON_OUTPUT); 
```

Comment:
```suggestion
          const input = JSON.parse(process.env.JSON); 
```

Reply:
oh yes, you're right, this is because I am testing it on another branch/repo 😅 oops